src/components/App.js

```diff
@@ -2,26 +2,43 @@ import React, {Component} from 'react';
 import Header from './Header';
 import MovieContainer from './MovieContainer';
 import MovieView from './MovieView';
-import movieData from '../movieData';
+import apiCalls from '../apiCalls';
+import { cleanMovieData, formatAverageRating } from "../utilities.js";
 import '../css/App.css';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData,
+      movies: [],
       error: "",
       isLoading: true,
       selectedMovie: ''
```

Comment:
Does the `selectedMovie` stay as a string, or does it become an object? Typically, we try to make the initial value in state the same datatype as its eventual value. (or you might see `null` used as a starting value)